### PR TITLE
feat: created tab basic design

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check

`src/App.js`
`src/components/maintab/MainTab.jsx`
`src/components/nameform/AskName.jsx`

## [Live](https://deploy-preview-1--resplendent-tiramisu-e2fd5e.netlify.app/)

In this PR, user can enter their name and focus for today, also edit them. 
If you have any better file name suggestion then please suggest because I struggled a lot finding good file name 😅
Thank you for reviewing my pr :)

### CODE REVIEW

**Review by Hamza-777 — APPROVED** [review]
Code looks clean, preview looks good too, just a question that, do we just have to make a react app or is it supposed to be an actual chrome extension?

**Comment by Developers-codz** [comment]
> Code looks clean, preview looks good too, just a question that, do we just have to make a react app or is it supposed to be an actual chrome extension?

No it will be a real chrome extension.

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,28 @@
...
-    </div>
+  const [flag, setFlag] = useState(localStorage.getItem(false));
+  // const funct = async () => {
+  //   try {
+  //     const response = await axios.get(
+  //       "https://api.unsplash.com/photos/random?client_id=AgBYk9WdFOk5290DJb2lZReoAvaQrmRjrrbvd3ay4TY&query=nature&orientation=landscape"
+  //     );
+  //     console.log(response.data.urls.raw);
+  //     setImage(response.data.urls.raw);
+  //   } catch (err) {
```
Comment (vishal804) [inline src/App.js:15]: can remove commented code if not needed

**Review by vishal804 — APPROVED** [review]

**Comment by vishal804** [comment]
Code Looks Clean and Good

File: src/App.js (hunk 1 above)
Comment (Developers-codz) [reply src/App.js:15]: Will remove it

**Pull request merged**